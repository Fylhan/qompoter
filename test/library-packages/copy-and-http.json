{
	"name": "install-action-for-library-packages/copy-and-http",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/acme": "v1.*-lib",
		"fylhan/qompoter-test-package4git": "v1.0-lib",
		"fylhan/qompoter-test-package": "v1.0-lib"
	},
	"repositories": {
		"fylhan/qompoter-test-package4git": { "url": "https://github.com/Fylhan/qompoter-test-package4git/releases/download/v1.0/qompoter-test-package4git-v1.0-lib.tar.gz" },
		"fylhan/qompoter-test-package": { "url": "https://wrong-uri.tar.gz" }
	}
}
