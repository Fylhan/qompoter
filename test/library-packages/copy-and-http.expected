Qompoter
======== install

* cp/acme v1.*-lib
  Downloading lib...
  Selected version: v1.1-lib
  done

* fylhan/qompoter-test-package4git v1.0-lib
  Downloading lib...
  Warning: no 'qompoter.pri' found for this package
  done

* fylhan/qompoter-test-package v1.0-lib
  Downloading lib...
  Warning: cannot find provided "https:///wrong-uri.tar.gz", let's try another repository
  Warning: no 'qompoter.pri' found for this package
  done

done
